# 4-1-ChIP-QCAlignment
Jakke Neiro$^1$
1. Aboobaker laboratory, Department of Zoology, University of Oxford

## Contents of notebook
* 1. Introduction
* 2. Data quality
    * 2.1 Data inspection
        * H3K27ac
        * H3K4me1
    * 2.2 FastQC H3K27ac
        * 2.2.1 X1-H3K27ace-_S12_L001_R1_001
        * 2.2.2 X1-H3K27ace-_S12_L001_R2_001
        * 2.2.3 X1-H3K27ace-_S12_L002_R1_001
        * 2.2.4 X1-H3K27ace-_S12_L002_R2_001
        * 2.2.5 X1-H3K27ace-_S12_L003_R1_001
        * 2.2.6 X1-H3K27ace-_S12_L003_R2_001
        * 2.2.7 X1-H3K27ace-_S12_L004_R1_001
        * 2.2.8 X1-H3K27ace-_S12_L004_R2_001
        * 2.2.9 X1-input-_S9_L001_R1_001
        * 2.2.10 X1-input-_S9_L001_R2_001
        * 2.2.11 X1-input-_S9_L002_R1_001
        * 2.2.12 X1-input-_S9_L002_R2_001
        * 2.2.13 X1-input-_S9_L003_R1_001
        * 2.2.14 X1-input-_S9_L003_R2_001
        * 2.2.15 X1-input-_S9_L004_R1_001
        * 2.2.16 X1-input-_S9_L004_R2_001
        * 2.2.17 x1_h3k27ac_rep2.R1
        * 2.2.18 x1_h3k27ac_rep2.R2
        * 2.2.19 x1_h3k27ac_rep3.R1
        * 2.2.20 x1_h3k27ac_rep3.R2
        * 2.2.21 x1_input_rep2.R1
        * 2.2.22 x1_input_rep2.R2
        * 2.2.23 x1_input_rep3_R1
        * 2.2.24 x1_input_rep3_R2
    * 2.3 FastQC H3K4me1
        * 2.3.1 h3k4me1_rep1_R1
        * 2.3.2 h3k4me1_rep1_R2
        * 2.3.3 h3k4me1_rep2_R1
        * 2.3.4 h3k4me1_rep2_R2
        * 2.3.5 input_rep1_R1
        * 2.3.6 input_rep1_R2
        * 2.3.7 input_rep2.R1
        * 2.3.8 input_rep2.R2
* 3. Trimming
    * 3.1 H3K27ac
        * 3.1.1 X1-H3K27ace-_S12_L001
        * 3.1.2 X1-H3K27ace-_S12_L002
        * 3.1.3 X1-H3K27ace-_S12_L003
        * 3.1.4 X1-H3K27ace-_S12_L004
        * 3.1.5 X1-input-_S9_L001
        * 3.1.6 X1-input-_S9_L002
        * 3.1.7 X1-input-_S9_L003
        * 3.1.8 X1-input-_S9_L004
    * 3.2 H3K4me1
        * 3.2.1 h3k4me1_rep1
        * 3.2.2 h3k4me1_rep2
        * 3.2.3 input_rep1
        * 3.2.4 input_rep2
* 4. Alignment
    * 4.1 H3K27ac
    * 4.2 H3K4me1

## Files
* Input: X1-H3K27ace-_S12_L001_R1_001.fastq.gz, X1-H3K27ace-_S12_L001_R2_001.fastq.gz, X1-H3K27ace-_S12_L002_R1_001.fastq.gz, X1-H3K27ace-_S12_L002_R2_001.fastq.gz, X1-H3K27ace-_S12_L003_R1_001.fastq.gz, X1-H3K27ace-_S12_L003_R2_001.fastq.gz, X1-H3K27ace-_S12_L004_R1_001.fastq.gz, X1-H3K27ace-_S12_L004_R2_001.fastq.gz, X1-input-_S9_L001_R1_001.fastq.gz, X1-input-_S9_L001_R2_001.fastq.gz,  X1-input-_S9_L002_R1_001.fastq.gz, X1-input-_S9_L002_R2_001.fastq.gz, X1-input-_S9_L003_R1_001.fastq.gz, X1-input-_S9_L003_R2_001.fastq.gz, X1-input-_S9_L004_R1_001.fastq.gz, X1-input-_S9_L004_R2_001.fastq.gz, x1_h3k27ac_rep2.R1.fastq.gz, x1_h3k27ac_rep2.R2.fastq.gz, x1_h3k27ac_rep3.R1.fastq.gz, x1_h3k27ac_rep3.R2.fastq.gz, x1_input_rep2.R1.fastq.gz, x1_input_rep2.R2.fastq.gz, x1_input_rep3_R1.fastq.gz, x1_input_rep3_R2.fastq.gz
* Output: X1-H3K27ace-_S12_L001_final.sam, X1-H3K27ace-_S12_L002_final.sam, X1-H3K27ace-_S12_L003_final.sam, X1-H3K27ace-_S12_L004_final.sam, X1-input-_S9_L001_final.sam, X1-input-_S9_L002_final.sam, X1-input-_S9_L003_final.sam, X1-input-_S9_L004_final.sam, x1_h3k27ac_rep2_final.sam, x1_input_rep2_final.sam, x1_h3k27ac_rep3_final.sam, x1_input_rep3_final.sam, h3k4me1_rep1_final.sam, input_rep1_final.sam, h3k4me1_rep2_final.sam, input_rep2_final.sam

# 1. Introduction

The H3K27ac and H3K4me1 ChIP-seq data was inspected, trimmed and aligned to the planarian genome. 

# 2. Data quality

## 2.1 Data inspection

The characteristics of the fastq files were inspected.

### 2.1.1 H3K27ac

In [18]:
%%bash
cd /hydra/chip_neiro/k27ac
ls *fastq.gz

X1-H3K27ace-_S12_L001_R1_001.fastq.gz
X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq.gz
X1-H3K27ace-_S12_L001_R2_001.fastq.gz
X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq.gz
X1-H3K27ace-_S12_L002_R1_001.fastq.gz
X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq.gz
X1-H3K27ace-_S12_L002_R2_001.fastq.gz
X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq.gz
X1-H3K27ace-_S12_L003_R1_001.fastq.gz
X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq.gz
X1-H3K27ace-_S12_L003_R2_001.fastq.gz
X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq.gz
X1-H3K27ace-_S12_L004_R1_001.fastq.gz
X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq.gz
X1-H3K27ace-_S12_L004_R2_001.fastq.gz
X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq.gz
X1-input-_S9_L001_R1_001.fastq.gz
X1-input-_S9_L001_R1_trimmomatic.fastq.gz
X1-input-_S9_L001_R2_001.fastq.gz
X1-input-_S9_L001_R2_trimmomatic.fastq.gz
X1-input-_S9_L002_R1_001.fastq.gz
X1-input-_S9_L002_R1_trimmomatic.fastq.gz
X1-input-_S9_L002_R2_001.fastq.gz
X1-input-_S9_L002_R2_trimmomatic.fastq.gz
X1-input-_S9_L003_R1_001

In [1]:
%%bash
cd /hydra/chip_neiro/k27ac
less k27ac.README.txt

These are three replicate X1 libraries for K27ac. K27ac is hypothesized to be a mark of active enhancers. 

rep1 was made by Anish on 13/12/2016. Raw reads are in this folder. 
rep2 and rep3 was made by Divya in 2018. Raw reads are in /data/Divya. 
Be careful not to confuse the Inputs between the corresponding Samples. 
The input for K27ac and K9me3 rep1 are the same - made by Anish. 
The inputs for K27ac and K9me3 rep2 and rep3 are the same - made by Divya. 


### 2.1.2 H3K4me1

In [2]:
%%bash
cd /hydra/chip_neiro
cd k4me1
less k4me1.README.txt

K4me1 is a mark of enhancers - regardless of active or inactive. 

These raw reads were generated by Yuli and are published. 

For visualization against ATAC BigWigs, to generate input normalized ChIP-seq tracks use: 

bamCompare -b1 xxx_repx.filtered.sorted.MARKDUP.bam -b2 input_repx.filtered.sorted.MARKDUP.bam -o xxx.input.repx.log2ratio.bw --scaleFactorsMethod SES --smoothLength 50 -bs 10


## 2.2 Fastqc H3K27ac

The quality of the sequencing results was checked with FastQC:

In [2]:
#%%bash
#cd ../chip_sexual_organised/k27ac
#fastqc X1-H3K27ace-_S12_L001_R1_001.fastq.gz X1-H3K27ace-_S12_L001_R2_001.fastq.gz X1-H3K27ace-_S12_L002_R1_001.fastq.gz X1-H3K27ace-_S12_L002_R2_001.fastq.gz X1-H3K27ace-_S12_L003_R1_001.fastq.gz X1-H3K27ace-_S12_L003_R2_001.fastq.gz X1-H3K27ace-_S12_L004_R1_001.fastq.gz X1-H3K27ace-_S12_L004_R2_001.fastq.gz

In [116]:
#%%bash
#cd ../chip_sexual_organised/k27ac
#fastqc X1-input-_S9_L001_R1_001.fastq.gz X1-input-_S9_L001_R2_001.fastq.gz X1-input-_S9_L002_R1_001.fastq.gz X1-input-_S9_L002_R2_001.fastq.gz X1-input-_S9_L003_R1_001.fastq.gz X1-input-_S9_L003_R2_001.fastq.gz X1-input-_S9_L004_R1_001.fastq.gz X1-input-_S9_L004_R2_001.fastq.gz

In [155]:
#%%bash
#cd ../chip_sexual_organised/k27ac
#fastqc x1_h3k27ac_rep2.R1.fastq.gz x1_h3k27ac_rep2.R2.fastq.gz x1_h3k27ac_rep3.R1.fastq.gz x1_h3k27ac_rep3.R2.fastq.gz

In [157]:
#%%bash
#cd ../chip_sexual_organised/k27ac
#fastqc  x1_input_rep2.R1.fastq.gz x1_input_rep2.R2.fastq.gz x1_input_rep3_R1.fastq.gz x1_input_rep3_R2.fastq.gz

### 2.2.1 X1-H3K27ace-_S12_L001_R1_001

Rep1 was made by Anish Datatni 13/12/2016.

In [7]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L001_R1_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L001_R1_001.fastq.gz
PASS	Per base sequence quality	X1-H3K27ace-_S12_L001_R1_001.fastq.gz
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L001_R1_001.fastq.gz
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L001_R1_001.fastq.gz
FAIL	Per base sequence content	X1-H3K27ace-_S12_L001_R1_001.fastq.gz
FAIL	Per sequence GC content	X1-H3K27ace-_S12_L001_R1_001.fastq.gz
PASS	Per base N content	X1-H3K27ace-_S12_L001_R1_001.fastq.gz
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L001_R1_001.fastq.gz
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L001_R1_001.fastq.gz
FAIL	Overrepresented sequences	X1-H3K27ace-_S12_L001_R1_001.fastq.gz
PASS	Adapter Content	X1-H3K27ace-_S12_L001_R1_001.fastq.gz


In [9]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L001_R1_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG	1146568	7.9896322813008815	TruSeq Adapter, Index 8 (100% over 38bp)
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	43664	0.3042639459070214	No Hit
>>END_MODULE


In [3]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-H3K27ace-_S12_L001_R1_001.fastq.gz|wc -l)/4|bc

14350698


### 2.2.2 X1-H3K27ace-_S12_L001_R2_001

Rep1 was made by Anish Dattani 13/12/2016.

In [14]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L001_R2_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L001_R2_001.fastq.gz
PASS	Per base sequence quality	X1-H3K27ace-_S12_L001_R2_001.fastq.gz
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L001_R2_001.fastq.gz
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L001_R2_001.fastq.gz
FAIL	Per base sequence content	X1-H3K27ace-_S12_L001_R2_001.fastq.gz
FAIL	Per sequence GC content	X1-H3K27ace-_S12_L001_R2_001.fastq.gz
PASS	Per base N content	X1-H3K27ace-_S12_L001_R2_001.fastq.gz
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L001_R2_001.fastq.gz
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L001_R2_001.fastq.gz
FAIL	Overrepresented sequences	X1-H3K27ace-_S12_L001_R2_001.fastq.gz
PASS	Adapter Content	X1-H3K27ace-_S12_L001_R2_001.fastq.gz


In [17]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L001_R2_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG	613075	4.272091852256942	No Hit
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	53395	0.372072494313517	No Hit
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGTGGG	25343	0.17659768186885405	No Hit


For L1:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG	1146568	7.9896322813008815	TruSeq Adapter, Index 8 (100% over 38bp)

In [4]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-H3K27ace-_S12_L001_R2_001.fastq.gz|wc -l)/4|bc

14350698


### 2.2.3 X1-H3K27ace-_S12_L002_R1_001

Rep1 was made by Anish on 13/12/2016.

In [20]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L002_R1_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L002_R1_001.fastq.gz
PASS	Per base sequence quality	X1-H3K27ace-_S12_L002_R1_001.fastq.gz
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L002_R1_001.fastq.gz
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L002_R1_001.fastq.gz
FAIL	Per base sequence content	X1-H3K27ace-_S12_L002_R1_001.fastq.gz
FAIL	Per sequence GC content	X1-H3K27ace-_S12_L002_R1_001.fastq.gz
PASS	Per base N content	X1-H3K27ace-_S12_L002_R1_001.fastq.gz
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L002_R1_001.fastq.gz
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L002_R1_001.fastq.gz
FAIL	Overrepresented sequences	X1-H3K27ace-_S12_L002_R1_001.fastq.gz
PASS	Adapter Content	X1-H3K27ace-_S12_L002_R1_001.fastq.gz


In [22]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L002_R1_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG	1117707	7.780274506858495	TruSeq Adapter, Index 8 (100% over 38bp)
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	42398	0.29512929465574295	No Hit
GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGC	14951	0.10407278844280421	TruSeq Adapter, Index 8 (97% over 38bp)


In [5]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-H3K27ace-_S12_L002_R1_001.fastq.gz|wc -l)/4|bc

14365907


### 2.2.4 X1-H3K27ace-_S12_L002_R2_001

Rep1 was made by Anish on 13/12/2016.

In [24]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L002_R2_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L002_R2_001.fastq.gz
PASS	Per base sequence quality	X1-H3K27ace-_S12_L002_R2_001.fastq.gz
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L002_R2_001.fastq.gz
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L002_R2_001.fastq.gz
FAIL	Per base sequence content	X1-H3K27ace-_S12_L002_R2_001.fastq.gz
FAIL	Per sequence GC content	X1-H3K27ace-_S12_L002_R2_001.fastq.gz
PASS	Per base N content	X1-H3K27ace-_S12_L002_R2_001.fastq.gz
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L002_R2_001.fastq.gz
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L002_R2_001.fastq.gz
FAIL	Overrepresented sequences	X1-H3K27ace-_S12_L002_R2_001.fastq.gz
PASS	Adapter Content	X1-H3K27ace-_S12_L002_R2_001.fastq.gz


In [26]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L002_R2_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG	522054	3.633978696924601	No Hit
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	51298	0.357081526422244	No Hit
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGTGGG	26708	0.1859123826988439	No Hit


For L2:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG	1117707	7.780274506858495	TruSeq Adapter, Index 8 (100% over 38bp)

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGC	14951	0.10407278844280421	TruSeq Adapter, Index 8 (97% over 38bp)

In [7]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-H3K27ace-_S12_L002_R2_001.fastq.gz|wc -l)/4|bc

14365907


### 2.2.5 X1-H3K27ace-_S12_L003_R1_001

Rep1 was made by Anish on 13/12/2016.

In [28]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L003_R1_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L003_R1_001.fastq.gz
PASS	Per base sequence quality	X1-H3K27ace-_S12_L003_R1_001.fastq.gz
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L003_R1_001.fastq.gz
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L003_R1_001.fastq.gz
FAIL	Per base sequence content	X1-H3K27ace-_S12_L003_R1_001.fastq.gz
FAIL	Per sequence GC content	X1-H3K27ace-_S12_L003_R1_001.fastq.gz
PASS	Per base N content	X1-H3K27ace-_S12_L003_R1_001.fastq.gz
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L003_R1_001.fastq.gz
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L003_R1_001.fastq.gz
FAIL	Overrepresented sequences	X1-H3K27ace-_S12_L003_R1_001.fastq.gz
PASS	Adapter Content	X1-H3K27ace-_S12_L003_R1_001.fastq.gz


In [32]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L003_R1_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG	1112403	7.83774447175936	TruSeq Adapter, Index 8 (100% over 38bp)
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	41787	0.2944219210496631	No Hit
GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGC	14474	0.10198058930463597	TruSeq Adapter, Index 8 (97% over 38bp)


In [9]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-H3K27ace-_S12_L003_R1_001.fastq.gz|wc -l)/4|bc

14192897


### 2.2.6 X1-H3K27ace-_S12_L003_R2_001

Rep1 was made by Anish on 13/12/2016.

In [34]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L003_R2_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L003_R2_001.fastq.gz
PASS	Per base sequence quality	X1-H3K27ace-_S12_L003_R2_001.fastq.gz
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L003_R2_001.fastq.gz
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L003_R2_001.fastq.gz
FAIL	Per base sequence content	X1-H3K27ace-_S12_L003_R2_001.fastq.gz
FAIL	Per sequence GC content	X1-H3K27ace-_S12_L003_R2_001.fastq.gz
PASS	Per base N content	X1-H3K27ace-_S12_L003_R2_001.fastq.gz
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L003_R2_001.fastq.gz
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L003_R2_001.fastq.gz
FAIL	Overrepresented sequences	X1-H3K27ace-_S12_L003_R2_001.fastq.gz
PASS	Adapter Content	X1-H3K27ace-_S12_L003_R2_001.fastq.gz


In [36]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L003_R2_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG	501908	3.5363322935409167	No Hit
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	45580	0.3211465566191314	No Hit
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGTGGGGGG	30079	0.21192995341261195	No Hit


For L3:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG	1112403	7.83774447175936	TruSeq Adapter, Index 8 (100% over 38bp)

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGC	14474	0.10198058930463597	TruSeq Adapter, Index 8 (97% over 38bp)

In [10]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-H3K27ace-_S12_L003_R2_001.fastq.gz|wc -l)/4|bc

14192897


### 2.2.7 X1-H3K27ace-_S12_L004_R1_001

Rep1 was made by Anish on 13/12/2016.

In [3]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L004_R1_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L004_R1_001.fastq.gz
PASS	Per base sequence quality	X1-H3K27ace-_S12_L004_R1_001.fastq.gz
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L004_R1_001.fastq.gz
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L004_R1_001.fastq.gz
FAIL	Per base sequence content	X1-H3K27ace-_S12_L004_R1_001.fastq.gz
FAIL	Per sequence GC content	X1-H3K27ace-_S12_L004_R1_001.fastq.gz
PASS	Per base N content	X1-H3K27ace-_S12_L004_R1_001.fastq.gz
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L004_R1_001.fastq.gz
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L004_R1_001.fastq.gz
FAIL	Overrepresented sequences	X1-H3K27ace-_S12_L004_R1_001.fastq.gz
PASS	Adapter Content	X1-H3K27ace-_S12_L004_R1_001.fastq.gz


In [5]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L004_R1_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG	1102332	7.6879541422628455	TruSeq Adapter, Index 8 (100% over 38bp)
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	40776	0.28438257993500127	No Hit
GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGC	14654	0.1022008614471137	TruSeq Adapter, Index 8 (97% over 38bp)


In [11]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-H3K27ace-_S12_L004_R1_001.fastq.gz|wc -l)/4|bc

14338431


### 2.2.8 X1-H3K27ace-_S12_L004_R2_001

Rep1 was made by Anish on 13/12/2016.

In [10]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L004_R2_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L004_R2_001.fastq.gz
PASS	Per base sequence quality	X1-H3K27ace-_S12_L004_R2_001.fastq.gz
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L004_R2_001.fastq.gz
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L004_R2_001.fastq.gz
FAIL	Per base sequence content	X1-H3K27ace-_S12_L004_R2_001.fastq.gz
FAIL	Per sequence GC content	X1-H3K27ace-_S12_L004_R2_001.fastq.gz
PASS	Per base N content	X1-H3K27ace-_S12_L004_R2_001.fastq.gz
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L004_R2_001.fastq.gz
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L004_R2_001.fastq.gz
FAIL	Overrepresented sequences	X1-H3K27ace-_S12_L004_R2_001.fastq.gz
PASS	Adapter Content	X1-H3K27ace-_S12_L004_R2_001.fastq.gz


In [12]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-H3K27ace-_S12_L004_R2_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG	486248	3.3912218149949602	No Hit
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	45790	0.3193515385330515	No Hit
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGTGGG	30073	0.20973703468671012	No Hit


For L4:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG	1102332	7.6879541422628455	TruSeq Adapter, Index 8 (100% over 38bp)

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGC	14654	0.1022008614471137	TruSeq Adapter, Index 8 (97% over 38bp)

For L1-4:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG TruSeq Adapter, Index 8 

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGC TruSeq Adapter, Index 8

### 2.2.9 X1-input-_S9_L001_R1_001

Rep1 was made by Anish on 13/12/2016.

In [16]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L001_R1_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L001_R1_001.fastq.gz
PASS	Per base sequence quality	X1-input-_S9_L001_R1_001.fastq.gz
PASS	Per tile sequence quality	X1-input-_S9_L001_R1_001.fastq.gz
PASS	Per sequence quality scores	X1-input-_S9_L001_R1_001.fastq.gz
FAIL	Per base sequence content	X1-input-_S9_L001_R1_001.fastq.gz
PASS	Per sequence GC content	X1-input-_S9_L001_R1_001.fastq.gz
PASS	Per base N content	X1-input-_S9_L001_R1_001.fastq.gz
WARN	Sequence Length Distribution	X1-input-_S9_L001_R1_001.fastq.gz
PASS	Sequence Duplication Levels	X1-input-_S9_L001_R1_001.fastq.gz
FAIL	Overrepresented sequences	X1-input-_S9_L001_R1_001.fastq.gz
PASS	Adapter Content	X1-input-_S9_L001_R1_001.fastq.gz


In [18]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L001_R1_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG	81539	1.7919481774714145	TruSeq Adapter, Index 7 (97% over 36bp)
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	6022	0.13234295152911932	No Hit
>>END_MODULE


In [13]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-input-_S9_L001_R1_001.fastq.gz|wc -l)/4|bc

4550299


### 2.2.10 X1-input-_S9_L001_R2_001

Rep1 was made by Anish on 13/12/2016.

In [21]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L001_R2_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L001_R2_001.fastq.gz
PASS	Per base sequence quality	X1-input-_S9_L001_R2_001.fastq.gz
PASS	Per tile sequence quality	X1-input-_S9_L001_R2_001.fastq.gz
PASS	Per sequence quality scores	X1-input-_S9_L001_R2_001.fastq.gz
FAIL	Per base sequence content	X1-input-_S9_L001_R2_001.fastq.gz
WARN	Per sequence GC content	X1-input-_S9_L001_R2_001.fastq.gz
PASS	Per base N content	X1-input-_S9_L001_R2_001.fastq.gz
WARN	Sequence Length Distribution	X1-input-_S9_L001_R2_001.fastq.gz
PASS	Sequence Duplication Levels	X1-input-_S9_L001_R2_001.fastq.gz
WARN	Overrepresented sequences	X1-input-_S9_L001_R2_001.fastq.gz
PASS	Adapter Content	X1-input-_S9_L001_R2_001.fastq.gz


In [23]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L001_R2_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG	44916	0.9870999685954703	No Hit
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	9860	0.21668905713668488	No Hit
>>END_MODULE


For L1:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG	81539	1.7919481774714145	TruSeq Adapter, Index 7 (97% over 36bp)

In [14]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-input-_S9_L001_R2_001.fastq.gz|wc -l)/4|bc

4550299


### 2.2.11 X1-input-_S9_L002_R1_001

Rep1 was made by Anish on 13/12/2016.

In [5]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L002_R1_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L002_R1_001.fastq.gz
PASS	Per base sequence quality	X1-input-_S9_L002_R1_001.fastq.gz
PASS	Per tile sequence quality	X1-input-_S9_L002_R1_001.fastq.gz
PASS	Per sequence quality scores	X1-input-_S9_L002_R1_001.fastq.gz
FAIL	Per base sequence content	X1-input-_S9_L002_R1_001.fastq.gz
PASS	Per sequence GC content	X1-input-_S9_L002_R1_001.fastq.gz
PASS	Per base N content	X1-input-_S9_L002_R1_001.fastq.gz
WARN	Sequence Length Distribution	X1-input-_S9_L002_R1_001.fastq.gz
PASS	Sequence Duplication Levels	X1-input-_S9_L002_R1_001.fastq.gz
FAIL	Overrepresented sequences	X1-input-_S9_L002_R1_001.fastq.gz
PASS	Adapter Content	X1-input-_S9_L002_R1_001.fastq.gz


In [6]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L002_R1_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG	77342	1.6975028872605418	TruSeq Adapter, Index 7 (97% over 36bp)
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	6143	0.13482661731583756	No Hit
>>END_MODULE


In [15]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-input-_S9_L002_R1_001.fastq.gz|wc -l)/4|bc

4556222


### 2.2.12 X1-input-_S9_L002_R2_001

Rep1 was made by Anish on 13/12/2016.

In [9]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L002_R2_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L002_R2_001.fastq.gz
PASS	Per base sequence quality	X1-input-_S9_L002_R2_001.fastq.gz
PASS	Per tile sequence quality	X1-input-_S9_L002_R2_001.fastq.gz
PASS	Per sequence quality scores	X1-input-_S9_L002_R2_001.fastq.gz
FAIL	Per base sequence content	X1-input-_S9_L002_R2_001.fastq.gz
WARN	Per sequence GC content	X1-input-_S9_L002_R2_001.fastq.gz
PASS	Per base N content	X1-input-_S9_L002_R2_001.fastq.gz
WARN	Sequence Length Distribution	X1-input-_S9_L002_R2_001.fastq.gz
PASS	Sequence Duplication Levels	X1-input-_S9_L002_R2_001.fastq.gz
WARN	Overrepresented sequences	X1-input-_S9_L002_R2_001.fastq.gz
PASS	Adapter Content	X1-input-_S9_L002_R2_001.fastq.gz


In [10]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L002_R2_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG	37226	0.8170365710889418	No Hit
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	9667	0.21217139990105838	No Hit
>>END_MODULE


For L2:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG	77342	1.6975028872605418	TruSeq Adapter, Index 7 (97% over 36bp)

In [16]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-input-_S9_L002_R2_001.fastq.gz|wc -l)/4|bc

4556222


### 2.2.13 X1-input-_S9_L003_R1_001

Rep1 was made by Anish on 13/12/2016.

In [13]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L003_R1_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L003_R1_001.fastq.gz
PASS	Per base sequence quality	X1-input-_S9_L003_R1_001.fastq.gz
PASS	Per tile sequence quality	X1-input-_S9_L003_R1_001.fastq.gz
PASS	Per sequence quality scores	X1-input-_S9_L003_R1_001.fastq.gz
FAIL	Per base sequence content	X1-input-_S9_L003_R1_001.fastq.gz
PASS	Per sequence GC content	X1-input-_S9_L003_R1_001.fastq.gz
PASS	Per base N content	X1-input-_S9_L003_R1_001.fastq.gz
WARN	Sequence Length Distribution	X1-input-_S9_L003_R1_001.fastq.gz
PASS	Sequence Duplication Levels	X1-input-_S9_L003_R1_001.fastq.gz
FAIL	Overrepresented sequences	X1-input-_S9_L003_R1_001.fastq.gz
PASS	Adapter Content	X1-input-_S9_L003_R1_001.fastq.gz


In [14]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L003_R1_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG	77057	1.7172433004624212	TruSeq Adapter, Index 7 (97% over 36bp)
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	5648	0.12586773636414286	No Hit
>>END_MODULE


In [17]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-input-_S9_L003_R1_001.fastq.gz|wc -l)/4|bc

4487250


### 2.2.14 X1-input-_S9_L003_R2_001

Rep1 was made by Anish on 13/12/2016.

In [17]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L003_R2_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L003_R2_001.fastq.gz
PASS	Per base sequence quality	X1-input-_S9_L003_R2_001.fastq.gz
PASS	Per tile sequence quality	X1-input-_S9_L003_R2_001.fastq.gz
PASS	Per sequence quality scores	X1-input-_S9_L003_R2_001.fastq.gz
FAIL	Per base sequence content	X1-input-_S9_L003_R2_001.fastq.gz
WARN	Per sequence GC content	X1-input-_S9_L003_R2_001.fastq.gz
PASS	Per base N content	X1-input-_S9_L003_R2_001.fastq.gz
WARN	Sequence Length Distribution	X1-input-_S9_L003_R2_001.fastq.gz
PASS	Sequence Duplication Levels	X1-input-_S9_L003_R2_001.fastq.gz
WARN	Overrepresented sequences	X1-input-_S9_L003_R2_001.fastq.gz
PASS	Adapter Content	X1-input-_S9_L003_R2_001.fastq.gz


In [18]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L003_R2_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG	36007	0.8024291046854978	No Hit
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	7386	0.16459969914758482	No Hit
>>END_MODULE


For L3:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG	77057	1.7172433004624212	TruSeq Adapter, Index 7 (97% over 36bp)

In [18]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-input-_S9_L003_R2_001.fastq.gz|wc -l)/4|bc

4487250


### 2.2.15 X1-input-_S9_L004_R1_001

Rep1 was made by Anish on 13/12/2016.

In [26]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L004_R1_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L004_R1_001.fastq.gz
PASS	Per base sequence quality	X1-input-_S9_L004_R1_001.fastq.gz
PASS	Per tile sequence quality	X1-input-_S9_L004_R1_001.fastq.gz
PASS	Per sequence quality scores	X1-input-_S9_L004_R1_001.fastq.gz
FAIL	Per base sequence content	X1-input-_S9_L004_R1_001.fastq.gz
PASS	Per sequence GC content	X1-input-_S9_L004_R1_001.fastq.gz
PASS	Per base N content	X1-input-_S9_L004_R1_001.fastq.gz
WARN	Sequence Length Distribution	X1-input-_S9_L004_R1_001.fastq.gz
PASS	Sequence Duplication Levels	X1-input-_S9_L004_R1_001.fastq.gz
FAIL	Overrepresented sequences	X1-input-_S9_L004_R1_001.fastq.gz
PASS	Adapter Content	X1-input-_S9_L004_R1_001.fastq.gz


In [27]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L004_R1_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG	75211	1.6555104807982268	TruSeq Adapter, Index 7 (97% over 36bp)
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	5680	0.12502558842368705	No Hit
>>END_MODULE


In [19]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-input-_S9_L004_R1_001.fastq.gz|wc -l)/4|bc

4543070


### 2.2.16 X1-input-_S9_L004_R2_001

Rep1 was made by Anish on 13/12/2016.

In [21]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L004_R2_001_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L004_R2_001.fastq.gz
PASS	Per base sequence quality	X1-input-_S9_L004_R2_001.fastq.gz
PASS	Per tile sequence quality	X1-input-_S9_L004_R2_001.fastq.gz
PASS	Per sequence quality scores	X1-input-_S9_L004_R2_001.fastq.gz
FAIL	Per base sequence content	X1-input-_S9_L004_R2_001.fastq.gz
WARN	Per sequence GC content	X1-input-_S9_L004_R2_001.fastq.gz
PASS	Per base N content	X1-input-_S9_L004_R2_001.fastq.gz
WARN	Sequence Length Distribution	X1-input-_S9_L004_R2_001.fastq.gz
PASS	Sequence Duplication Levels	X1-input-_S9_L004_R2_001.fastq.gz
WARN	Overrepresented sequences	X1-input-_S9_L004_R2_001.fastq.gz
PASS	Adapter Content	X1-input-_S9_L004_R2_001.fastq.gz


In [22]:
%%bash
cd ../chip_sexual_organised/k27ac
cd X1-input-_S9_L004_R2_001_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG	34257	0.7540495744067338	No Hit
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	7729	0.17012724875469673	No Hit
>>END_MODULE


For L4:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG	75211	1.6555104807982268	TruSeq Adapter, Index 7 (97% over 36bp)

For L1-4:
    
GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG	81539	1.7919481774714145	TruSeq Adapter, Index 7

In [20]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat X1-input-_S9_L004_R2_001.fastq.gz|wc -l)/4|bc

4543070


### 2.2.17 x1_h3k27ac_rep2.R1

rep2 and rep3 was made by Divya in 2018

In [161]:
%%bash
cd ../chip_sexual_organised/k27ac
cd x1_h3k27ac_rep2.R1_fastqc
less summary.txt

PASS	Basic Statistics	x1_h3k27ac_rep2.R1.fastq.gz
PASS	Per base sequence quality	x1_h3k27ac_rep2.R1.fastq.gz
PASS	Per tile sequence quality	x1_h3k27ac_rep2.R1.fastq.gz
PASS	Per sequence quality scores	x1_h3k27ac_rep2.R1.fastq.gz
PASS	Per base sequence content	x1_h3k27ac_rep2.R1.fastq.gz
PASS	Per sequence GC content	x1_h3k27ac_rep2.R1.fastq.gz
PASS	Per base N content	x1_h3k27ac_rep2.R1.fastq.gz
PASS	Sequence Length Distribution	x1_h3k27ac_rep2.R1.fastq.gz
PASS	Sequence Duplication Levels	x1_h3k27ac_rep2.R1.fastq.gz
PASS	Overrepresented sequences	x1_h3k27ac_rep2.R1.fastq.gz
PASS	Adapter Content	x1_h3k27ac_rep2.R1.fastq.gz


In [23]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat x1_h3k27ac_rep2.R1.fastq.gz|wc -l)/4|bc

27883302


### 2.2.18 x1_h3k27ac_rep2.R2

rep2 and rep3 was made by Divya in 2018

In [165]:
%%bash
cd ../chip_sexual_organised/k27ac
cd x1_h3k27ac_rep2.R2_fastqc
less summary.txt

PASS	Basic Statistics	x1_h3k27ac_rep2.R2.fastq.gz
PASS	Per base sequence quality	x1_h3k27ac_rep2.R2.fastq.gz
PASS	Per tile sequence quality	x1_h3k27ac_rep2.R2.fastq.gz
PASS	Per sequence quality scores	x1_h3k27ac_rep2.R2.fastq.gz
PASS	Per base sequence content	x1_h3k27ac_rep2.R2.fastq.gz
PASS	Per sequence GC content	x1_h3k27ac_rep2.R2.fastq.gz
PASS	Per base N content	x1_h3k27ac_rep2.R2.fastq.gz
PASS	Sequence Length Distribution	x1_h3k27ac_rep2.R2.fastq.gz
PASS	Sequence Duplication Levels	x1_h3k27ac_rep2.R2.fastq.gz
PASS	Overrepresented sequences	x1_h3k27ac_rep2.R2.fastq.gz
PASS	Adapter Content	x1_h3k27ac_rep2.R2.fastq.gz


In [24]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat x1_h3k27ac_rep2.R2.fastq.gz|wc -l)/4|bc

27883302


### 2.2.19 x1_h3k27ac_rep3.R1
rep2 and rep3 was made by Divya in 2018

In [170]:
%%bash
cd ../chip_sexual_organised/k27ac
cd x1_h3k27ac_rep3.R1_fastqc
less summary.txt

PASS	Basic Statistics	x1_h3k27ac_rep3.R1.fastq.gz
PASS	Per base sequence quality	x1_h3k27ac_rep3.R1.fastq.gz
PASS	Per tile sequence quality	x1_h3k27ac_rep3.R1.fastq.gz
PASS	Per sequence quality scores	x1_h3k27ac_rep3.R1.fastq.gz
PASS	Per base sequence content	x1_h3k27ac_rep3.R1.fastq.gz
PASS	Per sequence GC content	x1_h3k27ac_rep3.R1.fastq.gz
PASS	Per base N content	x1_h3k27ac_rep3.R1.fastq.gz
PASS	Sequence Length Distribution	x1_h3k27ac_rep3.R1.fastq.gz
PASS	Sequence Duplication Levels	x1_h3k27ac_rep3.R1.fastq.gz
PASS	Overrepresented sequences	x1_h3k27ac_rep3.R1.fastq.gz
PASS	Adapter Content	x1_h3k27ac_rep3.R1.fastq.gz


In [25]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat x1_h3k27ac_rep3.R1.fastq.gz|wc -l)/4|bc

29756249


### 2.2.20 x1_h3k27ac_rep3.R2

rep2 and rep3 was made by Divya in 2018

In [174]:
%%bash
cd ../chip_sexual_organised/k27ac
cd x1_h3k27ac_rep3.R2_fastqc
less summary.txt

PASS	Basic Statistics	x1_h3k27ac_rep3.R2.fastq.gz
PASS	Per base sequence quality	x1_h3k27ac_rep3.R2.fastq.gz
PASS	Per tile sequence quality	x1_h3k27ac_rep3.R2.fastq.gz
PASS	Per sequence quality scores	x1_h3k27ac_rep3.R2.fastq.gz
PASS	Per base sequence content	x1_h3k27ac_rep3.R2.fastq.gz
PASS	Per sequence GC content	x1_h3k27ac_rep3.R2.fastq.gz
PASS	Per base N content	x1_h3k27ac_rep3.R2.fastq.gz
PASS	Sequence Length Distribution	x1_h3k27ac_rep3.R2.fastq.gz
PASS	Sequence Duplication Levels	x1_h3k27ac_rep3.R2.fastq.gz
PASS	Overrepresented sequences	x1_h3k27ac_rep3.R2.fastq.gz
PASS	Adapter Content	x1_h3k27ac_rep3.R2.fastq.gz


In [26]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat x1_h3k27ac_rep3.R2.fastq.gz|wc -l)/4|bc

29756249


### 2.2.21 x1_input_rep2.R1

rep2 and rep3 was made by Divya in 2018

In [179]:
%%bash
cd ../chip_sexual_organised/k27ac
cd x1_input_rep2.R1_fastqc
less summary.txt

PASS	Basic Statistics	x1_input_rep2.R1.fastq.gz
PASS	Per base sequence quality	x1_input_rep2.R1.fastq.gz
PASS	Per tile sequence quality	x1_input_rep2.R1.fastq.gz
PASS	Per sequence quality scores	x1_input_rep2.R1.fastq.gz
PASS	Per base sequence content	x1_input_rep2.R1.fastq.gz
PASS	Per sequence GC content	x1_input_rep2.R1.fastq.gz
PASS	Per base N content	x1_input_rep2.R1.fastq.gz
PASS	Sequence Length Distribution	x1_input_rep2.R1.fastq.gz
PASS	Sequence Duplication Levels	x1_input_rep2.R1.fastq.gz
PASS	Overrepresented sequences	x1_input_rep2.R1.fastq.gz
PASS	Adapter Content	x1_input_rep2.R1.fastq.gz


In [27]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat x1_input_rep2.R1.fastq.gz|wc -l)/4|bc

29995548


### 2.2.22 x1_input_rep2.R2

rep2 and rep3 was made by Divya in 2018

In [182]:
%%bash
cd ../chip_sexual_organised/k27ac
cd x1_input_rep2.R2_fastqc
less summary.txt

PASS	Basic Statistics	x1_input_rep2.R2.fastq.gz
PASS	Per base sequence quality	x1_input_rep2.R2.fastq.gz
PASS	Per tile sequence quality	x1_input_rep2.R2.fastq.gz
PASS	Per sequence quality scores	x1_input_rep2.R2.fastq.gz
PASS	Per base sequence content	x1_input_rep2.R2.fastq.gz
PASS	Per sequence GC content	x1_input_rep2.R2.fastq.gz
PASS	Per base N content	x1_input_rep2.R2.fastq.gz
PASS	Sequence Length Distribution	x1_input_rep2.R2.fastq.gz
PASS	Sequence Duplication Levels	x1_input_rep2.R2.fastq.gz
PASS	Overrepresented sequences	x1_input_rep2.R2.fastq.gz
PASS	Adapter Content	x1_input_rep2.R2.fastq.gz


In [28]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat x1_input_rep2.R2.fastq.gz|wc -l)/4|bc

29995548


### 2.2.23 x1_input_rep3_R1

rep2 and rep3 was made by Divya in 2018

In [186]:
%%bash
cd ../chip_sexual_organised/k27ac
cd x1_input_rep3_R1_fastqc
less summary.txt

PASS	Basic Statistics	x1_input_rep3_R1.fastq.gz
PASS	Per base sequence quality	x1_input_rep3_R1.fastq.gz
PASS	Per tile sequence quality	x1_input_rep3_R1.fastq.gz
PASS	Per sequence quality scores	x1_input_rep3_R1.fastq.gz
PASS	Per base sequence content	x1_input_rep3_R1.fastq.gz
PASS	Per sequence GC content	x1_input_rep3_R1.fastq.gz
PASS	Per base N content	x1_input_rep3_R1.fastq.gz
PASS	Sequence Length Distribution	x1_input_rep3_R1.fastq.gz
PASS	Sequence Duplication Levels	x1_input_rep3_R1.fastq.gz
PASS	Overrepresented sequences	x1_input_rep3_R1.fastq.gz
PASS	Adapter Content	x1_input_rep3_R1.fastq.gz


In [34]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat x1_input_rep3_R1.fastq.gz|wc -l)/4|bc

39350607


### 2.2.24 x1_input_rep3_R2

rep2 and rep3 was made by Divya in 2018

In [189]:
%%bash
cd ../chip_sexual_organised/k27ac
cd x1_input_rep3_R2_fastqc
less summary.txt

PASS	Basic Statistics	x1_input_rep3_R2.fastq.gz
PASS	Per base sequence quality	x1_input_rep3_R2.fastq.gz
PASS	Per tile sequence quality	x1_input_rep3_R2.fastq.gz
PASS	Per sequence quality scores	x1_input_rep3_R2.fastq.gz
PASS	Per base sequence content	x1_input_rep3_R2.fastq.gz
PASS	Per sequence GC content	x1_input_rep3_R2.fastq.gz
PASS	Per base N content	x1_input_rep3_R2.fastq.gz
PASS	Sequence Length Distribution	x1_input_rep3_R2.fastq.gz
PASS	Sequence Duplication Levels	x1_input_rep3_R2.fastq.gz
PASS	Overrepresented sequences	x1_input_rep3_R2.fastq.gz
PASS	Adapter Content	x1_input_rep3_R2.fastq.gz


In [35]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k27ac
echo $(zcat x1_input_rep3_R2.fastq.gz|wc -l)/4|bc

39350607


### 2.1.25 Conclusions

Rep1 overrepresented sequences:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG TruSeq Adapter, Index 8 

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGC TruSeq Adapter, Index 8

Rep1 input overrepresented sequences:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG	81539 TruSeq Adapter, Index 7

Rep2 and 3 no overrepresnted sequences 

## 2.3 FastQC H3k4me1

These raw reads were generated by Yuli and are published. 

In [31]:
#%%bash
#cd ../chip_sexual_organised/k4me1
#fastqc h3k4me1_rep1_R1.fastq.gz h3k4me1_rep1_R2.fastq.gz h3k4me1_rep2_R1.fastq.gz h3k4me1_rep2_R2.fastq.gz

In [82]:
#%%bash
#cd ../chip_sexual_organised/k4me1
#fastqc input_rep1_R1.fastq.gz input_rep1_R2.fastq.gz input_rep2.R1.fastq.gz input_rep2.R2.fastq.gz

### 2.3.1 h3k4me1_rep1_R1

In [36]:
%%bash
cd ../chip_sexual_organised/k4me1
cd h3k4me1_rep1_R1_fastqc
less summary.txt

PASS	Basic Statistics	h3k4me1_rep1_R1.fastq.gz
PASS	Per base sequence quality	h3k4me1_rep1_R1.fastq.gz
PASS	Per sequence quality scores	h3k4me1_rep1_R1.fastq.gz
FAIL	Per base sequence content	h3k4me1_rep1_R1.fastq.gz
PASS	Per sequence GC content	h3k4me1_rep1_R1.fastq.gz
PASS	Per base N content	h3k4me1_rep1_R1.fastq.gz
WARN	Sequence Length Distribution	h3k4me1_rep1_R1.fastq.gz
PASS	Sequence Duplication Levels	h3k4me1_rep1_R1.fastq.gz
FAIL	Overrepresented sequences	h3k4me1_rep1_R1.fastq.gz
PASS	Adapter Content	h3k4me1_rep1_R1.fastq.gz


In [38]:
%%bash
cd ../chip_sexual_organised/k4me1
cd h3k4me1_rep1_R1_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GATCGGAAGAGCACACGTCTGAACTCCAGTCACCGATGT	1231626	3.940509670298028	TruSeq Adapter, Index 2 (100% over 39bp)
>>END_MODULE
>>Adapter Content	pass


In [39]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k4me1
echo $(zcat h3k4me1_rep1_R1.fastq.gz|wc -l)/4|bc

31255500


### 2.3.2 h3k4me1_rep1_R2

In [42]:
%%bash
cd ../chip_sexual_organised/k4me1
cd h3k4me1_rep1_R2_fastqc
less summary.txt

PASS	Basic Statistics	h3k4me1_rep1_R2.fastq.gz
PASS	Per base sequence quality	h3k4me1_rep1_R2.fastq.gz
PASS	Per sequence quality scores	h3k4me1_rep1_R2.fastq.gz
FAIL	Per base sequence content	h3k4me1_rep1_R2.fastq.gz
WARN	Per sequence GC content	h3k4me1_rep1_R2.fastq.gz
PASS	Per base N content	h3k4me1_rep1_R2.fastq.gz
WARN	Sequence Length Distribution	h3k4me1_rep1_R2.fastq.gz
PASS	Sequence Duplication Levels	h3k4me1_rep1_R2.fastq.gz
FAIL	Overrepresented sequences	h3k4me1_rep1_R2.fastq.gz
PASS	Adapter Content	h3k4me1_rep1_R2.fastq.gz


In [43]:
%%bash
cd ../chip_sexual_organised/k4me1
cd h3k4me1_rep1_R2_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG	1197065	3.829933931628034	No Hit
>>END_MODULE
>>Adapter Content	pass


For rep1:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACCGATGT	1231626	3.940509670298028	TruSeq Adapter, Index 2

In [40]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k4me1
echo $(zcat h3k4me1_rep1_R2.fastq.gz|wc -l)/4|bc

31255500


### 2.3.3 h3k4me1_rep2_R1

In [46]:
%%bash
cd ../chip_sexual_organised/k4me1
cd h3k4me1_rep2_R1_fastqc
less summary.txt

PASS	Basic Statistics	h3k4me1_rep2_R1.fastq.gz
PASS	Per base sequence quality	h3k4me1_rep2_R1.fastq.gz
PASS	Per sequence quality scores	h3k4me1_rep2_R1.fastq.gz
FAIL	Per base sequence content	h3k4me1_rep2_R1.fastq.gz
PASS	Per sequence GC content	h3k4me1_rep2_R1.fastq.gz
PASS	Per base N content	h3k4me1_rep2_R1.fastq.gz
WARN	Sequence Length Distribution	h3k4me1_rep2_R1.fastq.gz
PASS	Sequence Duplication Levels	h3k4me1_rep2_R1.fastq.gz
FAIL	Overrepresented sequences	h3k4me1_rep2_R1.fastq.gz
PASS	Adapter Content	h3k4me1_rep2_R1.fastq.gz


In [47]:
%%bash
cd ../chip_sexual_organised/k4me1
cd h3k4me1_rep2_R1_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GATCGGAAGAGCACACGTCTGAACTCCAGTCACTAGCTT	1316788	4.335010476645132	TruSeq Adapter, Index 10 (100% over 39bp)
>>END_MODULE
>>Adapter Content	pass


In [41]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k4me1
echo $(zcat h3k4me1_rep2_R1.fastq.gz|wc -l)/4|bc

30375659


### 2.3.4 h3k4me1_rep2_R2

In [51]:
%%bash
cd ../chip_sexual_organised/k4me1
cd h3k4me1_rep2_R2_fastqc
less summary.txt

PASS	Basic Statistics	h3k4me1_rep2_R2.fastq.gz
PASS	Per base sequence quality	h3k4me1_rep2_R2.fastq.gz
PASS	Per sequence quality scores	h3k4me1_rep2_R2.fastq.gz
FAIL	Per base sequence content	h3k4me1_rep2_R2.fastq.gz
FAIL	Per sequence GC content	h3k4me1_rep2_R2.fastq.gz
PASS	Per base N content	h3k4me1_rep2_R2.fastq.gz
WARN	Sequence Length Distribution	h3k4me1_rep2_R2.fastq.gz
PASS	Sequence Duplication Levels	h3k4me1_rep2_R2.fastq.gz
FAIL	Overrepresented sequences	h3k4me1_rep2_R2.fastq.gz
PASS	Adapter Content	h3k4me1_rep2_R2.fastq.gz


In [52]:
%%bash
cd ../chip_sexual_organised/k4me1
cd h3k4me1_rep2_R2_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG	1287956	4.240092371329294	No Hit
>>END_MODULE
>>Adapter Content	pass


For rep2:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACTAGCTT	1316788	4.335010476645132	TruSeq Adapter, Index 10 (100% over 39bp)

In [42]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k4me1
echo $(zcat h3k4me1_rep2_R2.fastq.gz|wc -l)/4|bc

30375659


### 2.3.5 input_rep1_R1

In [60]:
%%bash
cd ../chip_sexual_organised/k4me1
cd input_rep1_R1_fastqc
less summary.txt

PASS	Basic Statistics	input_rep1_R1.fastq.gz
PASS	Per base sequence quality	input_rep1_R1.fastq.gz
PASS	Per sequence quality scores	input_rep1_R1.fastq.gz
FAIL	Per base sequence content	input_rep1_R1.fastq.gz
PASS	Per sequence GC content	input_rep1_R1.fastq.gz
PASS	Per base N content	input_rep1_R1.fastq.gz
WARN	Sequence Length Distribution	input_rep1_R1.fastq.gz
PASS	Sequence Duplication Levels	input_rep1_R1.fastq.gz
FAIL	Overrepresented sequences	input_rep1_R1.fastq.gz
PASS	Adapter Content	input_rep1_R1.fastq.gz


In [66]:
%%bash
cd ../chip_sexual_organised/k4me1
cd input_rep1_R1_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGA	1534234	4.698910117127738	TruSeq Adapter, Index 8 (100% over 39bp)
>>END_MODULE
>>Adapter Content	pass


In [43]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k4me1
echo $(zcat input_rep1_R1.fastq.gz|wc -l)/4|bc

32650848


### 2.3.6 input_rep1_R2

In [65]:
%%bash
cd ../chip_sexual_organised/k4me1
cd input_rep1_R2_fastqc
less summary.txt

PASS	Basic Statistics	input_rep1_R2.fastq.gz
PASS	Per base sequence quality	input_rep1_R2.fastq.gz
PASS	Per sequence quality scores	input_rep1_R2.fastq.gz
FAIL	Per base sequence content	input_rep1_R2.fastq.gz
FAIL	Per sequence GC content	input_rep1_R2.fastq.gz
PASS	Per base N content	input_rep1_R2.fastq.gz
WARN	Sequence Length Distribution	input_rep1_R2.fastq.gz
PASS	Sequence Duplication Levels	input_rep1_R2.fastq.gz
FAIL	Overrepresented sequences	input_rep1_R2.fastq.gz
PASS	Adapter Content	input_rep1_R2.fastq.gz


In [67]:
%%bash
cd ../chip_sexual_organised/k4me1
cd input_rep1_R2_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG	1490936	4.566301003882043	No Hit
>>END_MODULE
>>Adapter Content	pass


For rep1 input:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGA	1534234	4.698910117127738	TruSeq Adapter, Index 8 (100% over 39bp)

In [44]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k4me1
echo $(zcat input_rep1_R2.fastq.gz|wc -l)/4|bc

32650848


### 2.3.7 input_rep2.R1

In [73]:
%%bash
cd ../chip_sexual_organised/k4me1
cd input_rep2.R1_fastqc
less summary.txt

PASS	Basic Statistics	input_rep2.R1.fastq.gz
PASS	Per base sequence quality	input_rep2.R1.fastq.gz
PASS	Per sequence quality scores	input_rep2.R1.fastq.gz
FAIL	Per base sequence content	input_rep2.R1.fastq.gz
PASS	Per sequence GC content	input_rep2.R1.fastq.gz
PASS	Per base N content	input_rep2.R1.fastq.gz
WARN	Sequence Length Distribution	input_rep2.R1.fastq.gz
PASS	Sequence Duplication Levels	input_rep2.R1.fastq.gz
FAIL	Overrepresented sequences	input_rep2.R1.fastq.gz
PASS	Adapter Content	input_rep2.R1.fastq.gz


In [74]:
%%bash
cd ../chip_sexual_organised/k4me1
cd input_rep2.R1_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCA	918344	3.4431290053092054	TruSeq Adapter, Index 4 (100% over 39bp)
CATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCA	103628	0.38853041187418047	TruSeq Adapter, Index 4 (97% over 39bp)
>>END_MODULE


In [45]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k4me1
echo $(zcat input_rep2.R1.fastq.gz|wc -l)/4|bc

26671786


### 2.3.8 input_rep2.R2

In [77]:
%%bash
cd ../chip_sexual_organised/k4me1
cd input_rep2.R2_fastqc
less summary.txt

PASS	Basic Statistics	input_rep2.R2.fastq.gz
PASS	Per base sequence quality	input_rep2.R2.fastq.gz
PASS	Per sequence quality scores	input_rep2.R2.fastq.gz
FAIL	Per base sequence content	input_rep2.R2.fastq.gz
FAIL	Per sequence GC content	input_rep2.R2.fastq.gz
PASS	Per base N content	input_rep2.R2.fastq.gz
WARN	Sequence Length Distribution	input_rep2.R2.fastq.gz
PASS	Sequence Duplication Levels	input_rep2.R2.fastq.gz
FAIL	Overrepresented sequences	input_rep2.R2.fastq.gz
PASS	Adapter Content	input_rep2.R2.fastq.gz


In [78]:
%%bash
cd ../chip_sexual_organised/k4me1
cd input_rep2.R2_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	fail
#Sequence	Count	Percentage	Possible Source
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG	834923	3.130360299081584	No Hit
>>END_MODULE
>>Adapter Content	pass


For rep2 input:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCA	918344	3.4431290053092054	TruSeq Adapter, Index 4 (100% over 39bp)
CATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCA	103628	0.38853041187418047	TruSeq Adapter, Index 4 (97% over 39bp)

In [46]:
%%bash
cd ../../../../../data/ngs/smed/facs_rnaseq_datasets/sexual/chip_neiro/k4me1
echo $(zcat input_rep2.R2.fastq.gz|wc -l)/4|bc

26671786


### Conclusions

rep1: GATCGGAAGAGCACACGTCTGAACTCCAGTCACCGATGT	1231626	3.940509670298028	TruSeq Adapter, Index 2

rep2: GATCGGAAGAGCACACGTCTGAACTCCAGTCACTAGCTT	1316788	4.335010476645132	TruSeq Adapter, Index 10 (100% over 39bp)

rep1 input: GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGA 1534234 4.698910117127738 TruSeq Adapter, Index 8 (100% over 39bp)

rep2 input:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCA 918344 3.4431290053092054 TruSeq Adapter, Index 4 (100% over 39bp)

CATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCA 103628 0.38853041187418047 TruSeq Adapter, Index 4 (97% over 39bp)

# 3. Trimming

**H3K27ac**

Rep1 overrepresented sequences:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG TruSeq Adapter, Index 8 

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGC TruSeq Adapter, Index 8

Rep1 input overrepresented sequences:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG	81539 TruSeq Adapter, Index 7

Rep2 and 3 no overrepresnted sequences 

**H3K9me3**
X1-input-_S9_allLanes_R1_001: GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG TruSeq Adapter, Index 7

X1-H3K9me3-_S10_allLanes_R1_001: GATCGGAAGAGCACACGTCTGAACTCCAGTCACATCACG TruSeq Adapter, Index 1


GATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCA TruSeq Adapter, Index 4

CATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCA TruSeq Adapter, Index 4

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGA TruSeq Adapter, Index 8

GATCGGAAGAGCACACGTCTGAACTCCAGTCACTAGCTT TruSeq Adapter, Index 10

GATCGGAAGAGCACACGTCTGAACTCCAGTCACCGATGT TruSeq Adapter, Index 2 

GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG TruSeq Adapter, Index 7 

GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG	TruSeq Adapter, Index 7

GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG TruSeq Adapter, Index 8

## 3.1 H3K27ac

In [128]:
%%bash
cd ../chip_sexual_organised/k27ac/
echo ">TrueSeq8" > TrueSeq8.fa
echo "GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG" >> TrueSeq8.fa
less TrueSeq8.fa

>TrueSeq8
GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTG


In [25]:
%%bash
cd ../chip_sexual_organised/k27ac/
echo ">TrueSeq7" > TrueSeq7.fa
echo "GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG" >> TrueSeq7.fa
less TrueSeq7.fa

>TrueSeq7
GATCGGAAGAGCACACGTCTGAACTCCAGTCACGAGTGG


### 3.1.1 X1-H3K27ace-_S12_L001

Trimming:

In [144]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#java -jar ~/software/Trimmomatic-0.39/trimmomatic-0.39.jar PE X1-H3K27ace-_S12_L001_R1_001.fastq.gz X1-H3K27ace-_S12_L001_R2_001.fastq.gz X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq X1-H3K27ace-_S12_L001_R1_untrimmomatic.fastq X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq X1-H3K27ace-_S12_L001_R2_untrimmomatic.fastq ILLUMINACLIP:TrueSeq8.fa:2:30:10

In [139]:
%%bash
cd ../chip_sexual_organised/k27ac/
grep -c "@" X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq
grep -c "@" X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq

13017500
13017500


Quality checking:

In [9]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#fastqc X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq

**X1-H3K27ace-_S12_L001_R1_trimmomatic**

In [14]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L001_R1_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq
PASS	Per base sequence quality	X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq
FAIL	Per base sequence content	X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq
WARN	Per sequence GC content	X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq
PASS	Per base N content	X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq
WARN	Overrepresented sequences	X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq
PASS	Adapter Content	X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq


In [15]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L001_R1_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	43664	0.3354253888995583	No Hit
>>END_MODULE
>>Adapter Content	pass


**X1-H3K27ace-_S12_L001_R2_trimmomatic**

In [20]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L001_R2_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq
PASS	Per base sequence quality	X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq
FAIL	Per base sequence content	X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq
PASS	Per sequence GC content	X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq
PASS	Per base N content	X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq
WARN	Overrepresented sequences	X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq
PASS	Adapter Content	X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq


In [21]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L001_R2_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	52310	0.4018436719800269	No Hit
>>END_MODULE
>>Adapter Content	pass


### 3.1.2 X1-H3K27ace-_S12_L002

Trimming:

In [143]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#java -jar ~/software/Trimmomatic-0.39/trimmomatic-0.39.jar PE X1-H3K27ace-_S12_L002_R1_001.fastq.gz X1-H3K27ace-_S12_L002_R2_001.fastq.gz X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq X1-H3K27ace-_S12_L002_R1_untrimmomatic.fastq X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq X1-H3K27ace-_S12_L002_R2_untrimmomatic.fastq ILLUMINACLIP:TrueSeq8.fa:2:30:10

In [146]:
%%bash
cd ../chip_sexual_organised/k27ac/
grep -c "@" X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq
grep -c "@" X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq

13059617
13059617


Quality checking:

In [26]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#fastqc X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq

**X1-H3K27ace-_S12_L002_R1_trimmomatic**

In [32]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L002_R1_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq
PASS	Per base sequence quality	X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq
FAIL	Per base sequence content	X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq
WARN	Per sequence GC content	X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq
PASS	Per base N content	X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq
WARN	Overrepresented sequences	X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq
PASS	Adapter Content	X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq


In [33]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L002_R1_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	42398	0.3246496432475776	No Hit
>>END_MODULE
>>Adapter Content	pass


**X1-H3K27ace-_S12_L002_R2_trimmomatic**

In [35]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L002_R2_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq
PASS	Per base sequence quality	X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq
FAIL	Per base sequence content	X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq
PASS	Per sequence GC content	X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq
PASS	Per base N content	X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq
WARN	Overrepresented sequences	X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq
PASS	Adapter Content	X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq


In [36]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L002_R2_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	50283	0.38502660529784294	No Hit
>>END_MODULE
>>Adapter Content	pass


### 3.1.3 X1-H3K27ace-_S12_L003

Trimming:

In [148]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#java -jar ~/software/Trimmomatic-0.39/trimmomatic-0.39.jar PE X1-H3K27ace-_S12_L003_R1_001.fastq.gz X1-H3K27ace-_S12_L003_R2_001.fastq.gz X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq X1-H3K27ace-_S12_L003_R1_untrimmomatic.fastq X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq X1-H3K27ace-_S12_L003_R2_untrimmomatic.fastq ILLUMINACLIP:TrueSeq8.fa:2:30:10

In [149]:
%%bash
cd ../chip_sexual_organised/k27ac/
grep -c "@" X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq
grep -c "@" X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq

12875042
12875042


In [39]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#fastqc X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq

**X1-H3K27ace-_S12_L003_R1_trimmomatic**

In [42]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L003_R1_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq
PASS	Per base sequence quality	X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq
FAIL	Per base sequence content	X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq
WARN	Per sequence GC content	X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq
PASS	Per base N content	X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq
WARN	Overrepresented sequences	X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq
PASS	Adapter Content	X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq


In [43]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L003_R1_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	41787	0.3245581645481234	No Hit
>>END_MODULE
>>Adapter Content	pass


**X1-H3K27ace-_S12_L003_R2_trimmomatic**

In [46]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L003_R2_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq
PASS	Per base sequence quality	X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq
FAIL	Per base sequence content	X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq
PASS	Per sequence GC content	X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq
PASS	Per base N content	X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq
WARN	Overrepresented sequences	X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq
PASS	Adapter Content	X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq


In [47]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L003_R2_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	44850	0.3483483782033488	No Hit
>>END_MODULE
>>Adapter Content	pass


### 3.1.4 X1-H3K27ace-_S12_L004

Trimming:

In [151]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#java -jar ~/software/Trimmomatic-0.39/trimmomatic-0.39.jar PE X1-H3K27ace-_S12_L004_R1_001.fastq.gz X1-H3K27ace-_S12_L004_R2_001.fastq.gz X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq X1-H3K27ace-_S12_L004_R1_untrimmomatic.fastq X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq X1-H3K27ace-_S12_L004_R2_untrimmomatic.fastq ILLUMINACLIP:TrueSeq8.fa:2:30:10

In [152]:
%%bash
cd ../chip_sexual_organised/k27ac/
grep -c "@" X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq
grep -c "@" X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq

13039325
13039325


In [63]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#fastqc X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq

**X1-H3K27ace-_S12_L004_R1_trimmomatic**

In [70]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L004_R1_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq
PASS	Per base sequence quality	X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq
FAIL	Per base sequence content	X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq
WARN	Per sequence GC content	X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq
PASS	Per base N content	X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq
WARN	Overrepresented sequences	X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq
PASS	Adapter Content	X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq


In [73]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L004_R1_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	40776	0.31271557385064025	No Hit
>>END_MODULE
>>Adapter Content	pass


**X1-H3K27ace-_S12_L004_R2_trimmomatic**

In [71]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L004_R2_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq
PASS	Per base sequence quality	X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq
PASS	Per tile sequence quality	X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq
PASS	Per sequence quality scores	X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq
FAIL	Per base sequence content	X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq
PASS	Per sequence GC content	X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq
PASS	Per base N content	X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq
WARN	Overrepresented sequences	X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq
PASS	Adapter Content	X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq


In [75]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-H3K27ace-_S12_L004_R2_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	44986	0.34500252121946495	No Hit
>>END_MODULE
>>Adapter Content	pass


### 3.1.5 X1-input-_S9_L001

In [56]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#java -jar ~/software/Trimmomatic-0.39/trimmomatic-0.39.jar PE X1-input-_S9_L001_R1_001.fastq.gz X1-input-_S9_L001_R2_001.fastq.gz X1-input-_S9_L001_R1_trimmomatic.fastq X1-input-_S9_L001_R1_untrimmomatic.fastq X1-input-_S9_L001_R2_trimmomatic.fastq X1-input-_S9_L001_R2_untrimmomatic.fastq ILLUMINACLIP:TrueSeq7.fa:2:30:10

In [77]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#fastqc X1-input-_S9_L001_R1_trimmomatic.fastq X1-input-_S9_L001_R2_trimmomatic.fastq

**X1-input-_S9_L001_R1_trimmomatic**

In [80]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L001_R1_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L001_R1_trimmomatic.fastq
PASS	Per base sequence quality	X1-input-_S9_L001_R1_trimmomatic.fastq
PASS	Per tile sequence quality	X1-input-_S9_L001_R1_trimmomatic.fastq
PASS	Per sequence quality scores	X1-input-_S9_L001_R1_trimmomatic.fastq
FAIL	Per base sequence content	X1-input-_S9_L001_R1_trimmomatic.fastq
PASS	Per sequence GC content	X1-input-_S9_L001_R1_trimmomatic.fastq
PASS	Per base N content	X1-input-_S9_L001_R1_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-input-_S9_L001_R1_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-input-_S9_L001_R1_trimmomatic.fastq
WARN	Overrepresented sequences	X1-input-_S9_L001_R1_trimmomatic.fastq
PASS	Adapter Content	X1-input-_S9_L001_R1_trimmomatic.fastq


In [81]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L001_R1_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	6022	0.13517126144954678	No Hit
>>END_MODULE
>>Adapter Content	pass


**X1-input-_S9_L001_R2_trimmomatic**

In [85]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L001_R2_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L001_R2_trimmomatic.fastq
PASS	Per base sequence quality	X1-input-_S9_L001_R2_trimmomatic.fastq
PASS	Per tile sequence quality	X1-input-_S9_L001_R2_trimmomatic.fastq
PASS	Per sequence quality scores	X1-input-_S9_L001_R2_trimmomatic.fastq
FAIL	Per base sequence content	X1-input-_S9_L001_R2_trimmomatic.fastq
PASS	Per sequence GC content	X1-input-_S9_L001_R2_trimmomatic.fastq
PASS	Per base N content	X1-input-_S9_L001_R2_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-input-_S9_L001_R2_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-input-_S9_L001_R2_trimmomatic.fastq
WARN	Overrepresented sequences	X1-input-_S9_L001_R2_trimmomatic.fastq
PASS	Adapter Content	X1-input-_S9_L001_R2_trimmomatic.fastq


In [86]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L001_R1_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	6022	0.13517126144954678	No Hit
>>END_MODULE
>>Adapter Content	pass


### 3.1.6 X1-input-_S9_L002

In [57]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#java -jar ~/software/Trimmomatic-0.39/trimmomatic-0.39.jar PE X1-input-_S9_L002_R1_001.fastq.gz X1-input-_S9_L002_R2_001.fastq.gz X1-input-_S9_L002_R1_trimmomatic.fastq X1-input-_S9_L002_R1_untrimmomatic.fastq X1-input-_S9_L002_R2_trimmomatic.fastq X1-input-_S9_L002_R2_untrimmomatic.fastq ILLUMINACLIP:TrueSeq7.fa:2:30:10

In [88]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#fastqc X1-input-_S9_L002_R1_trimmomatic.fastq X1-input-_S9_L002_R2_trimmomatic.fastq

In [91]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L002_R1_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L002_R1_trimmomatic.fastq
PASS	Per base sequence quality	X1-input-_S9_L002_R1_trimmomatic.fastq
PASS	Per tile sequence quality	X1-input-_S9_L002_R1_trimmomatic.fastq
PASS	Per sequence quality scores	X1-input-_S9_L002_R1_trimmomatic.fastq
FAIL	Per base sequence content	X1-input-_S9_L002_R1_trimmomatic.fastq
PASS	Per sequence GC content	X1-input-_S9_L002_R1_trimmomatic.fastq
PASS	Per base N content	X1-input-_S9_L002_R1_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-input-_S9_L002_R1_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-input-_S9_L002_R1_trimmomatic.fastq
WARN	Overrepresented sequences	X1-input-_S9_L002_R1_trimmomatic.fastq
PASS	Adapter Content	X1-input-_S9_L002_R1_trimmomatic.fastq


In [92]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L002_R1_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	6143	0.13758235792376286	No Hit
>>END_MODULE
>>Adapter Content	pass


### 3.1.7  X1-input-_S9_L003

In [59]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#java -jar ~/software/Trimmomatic-0.39/trimmomatic-0.39.jar PE X1-input-_S9_L003_R1_001.fastq.gz X1-input-_S9_L003_R2_001.fastq.gz X1-input-_S9_L003_R1_trimmomatic.fastq X1-input-_S9_L003_R1_untrimmomatic.fastq X1-input-_S9_L003_R2_trimmomatic.fastq X1-input-_S9_L003_R2_untrimmomatic.fastq ILLUMINACLIP:TrueSeq7.fa:2:30:10

In [94]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#fastqc X1-input-_S9_L003_R1_trimmomatic.fastq X1-input-_S9_L003_R2_trimmomatic.fastq

In [97]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L003_R1_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L003_R1_trimmomatic.fastq
PASS	Per base sequence quality	X1-input-_S9_L003_R1_trimmomatic.fastq
PASS	Per tile sequence quality	X1-input-_S9_L003_R1_trimmomatic.fastq
PASS	Per sequence quality scores	X1-input-_S9_L003_R1_trimmomatic.fastq
FAIL	Per base sequence content	X1-input-_S9_L003_R1_trimmomatic.fastq
PASS	Per sequence GC content	X1-input-_S9_L003_R1_trimmomatic.fastq
PASS	Per base N content	X1-input-_S9_L003_R1_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-input-_S9_L003_R1_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-input-_S9_L003_R1_trimmomatic.fastq
WARN	Overrepresented sequences	X1-input-_S9_L003_R1_trimmomatic.fastq
PASS	Adapter Content	X1-input-_S9_L003_R1_trimmomatic.fastq


In [98]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L003_R1_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	5648	0.12850048938483122	No Hit
>>END_MODULE
>>Adapter Content	pass


In [102]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L003_R2_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L003_R2_trimmomatic.fastq
PASS	Per base sequence quality	X1-input-_S9_L003_R2_trimmomatic.fastq
PASS	Per tile sequence quality	X1-input-_S9_L003_R2_trimmomatic.fastq
PASS	Per sequence quality scores	X1-input-_S9_L003_R2_trimmomatic.fastq
FAIL	Per base sequence content	X1-input-_S9_L003_R2_trimmomatic.fastq
PASS	Per sequence GC content	X1-input-_S9_L003_R2_trimmomatic.fastq
PASS	Per base N content	X1-input-_S9_L003_R2_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-input-_S9_L003_R2_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-input-_S9_L003_R2_trimmomatic.fastq
WARN	Overrepresented sequences	X1-input-_S9_L003_R2_trimmomatic.fastq
PASS	Adapter Content	X1-input-_S9_L003_R2_trimmomatic.fastq


In [103]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L003_R2_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	7346	0.16713254161136157	No Hit
>>END_MODULE
>>Adapter Content	pass


### 3.1.8 X1-input-_S9_L004

In [61]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#java -jar ~/software/Trimmomatic-0.39/trimmomatic-0.39.jar PE X1-input-_S9_L004_R1_001.fastq.gz X1-input-_S9_L004_R2_001.fastq.gz X1-input-_S9_L004_R1_trimmomatic.fastq X1-input-_S9_L004_R1_untrimmomatic.fastq X1-input-_S9_L004_R2_trimmomatic.fastq X1-input-_S9_L004_R2_untrimmomatic.fastq ILLUMINACLIP:TrueSeq7.fa:2:30:10

In [105]:
#%%bash
#cd ../chip_sexual_organised/k27ac/
#fastqc X1-input-_S9_L004_R1_trimmomatic.fastq X1-input-_S9_L004_R2_trimmomatic.fastq

In [109]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L004_R1_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L004_R1_trimmomatic.fastq
PASS	Per base sequence quality	X1-input-_S9_L004_R1_trimmomatic.fastq
PASS	Per tile sequence quality	X1-input-_S9_L004_R1_trimmomatic.fastq
PASS	Per sequence quality scores	X1-input-_S9_L004_R1_trimmomatic.fastq
FAIL	Per base sequence content	X1-input-_S9_L004_R1_trimmomatic.fastq
PASS	Per sequence GC content	X1-input-_S9_L004_R1_trimmomatic.fastq
PASS	Per base N content	X1-input-_S9_L004_R1_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-input-_S9_L004_R1_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-input-_S9_L004_R1_trimmomatic.fastq
WARN	Overrepresented sequences	X1-input-_S9_L004_R1_trimmomatic.fastq
PASS	Adapter Content	X1-input-_S9_L004_R1_trimmomatic.fastq


In [110]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L004_R1_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	5680	0.12754855715185867	No Hit
>>END_MODULE
>>Adapter Content	pass


In [113]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L004_R2_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	X1-input-_S9_L004_R2_trimmomatic.fastq
PASS	Per base sequence quality	X1-input-_S9_L004_R2_trimmomatic.fastq
PASS	Per tile sequence quality	X1-input-_S9_L004_R2_trimmomatic.fastq
PASS	Per sequence quality scores	X1-input-_S9_L004_R2_trimmomatic.fastq
FAIL	Per base sequence content	X1-input-_S9_L004_R2_trimmomatic.fastq
PASS	Per sequence GC content	X1-input-_S9_L004_R2_trimmomatic.fastq
PASS	Per base N content	X1-input-_S9_L004_R2_trimmomatic.fastq
WARN	Sequence Length Distribution	X1-input-_S9_L004_R2_trimmomatic.fastq
PASS	Sequence Duplication Levels	X1-input-_S9_L004_R2_trimmomatic.fastq
WARN	Overrepresented sequences	X1-input-_S9_L004_R2_trimmomatic.fastq
PASS	Adapter Content	X1-input-_S9_L004_R2_trimmomatic.fastq


In [114]:
%%bash
cd ../chip_sexual_organised/k27ac/
cd X1-input-_S9_L004_R2_trimmomatic_fastqc
grep -A 4 "Overrepresented" fastqc_data.txt

>>Overrepresented sequences	warn
#Sequence	Count	Percentage	Possible Source
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN	7692	0.1727294897204396	No Hit
>>END_MODULE
>>Adapter Content	pass


## 3.2 H3K4me1

**H3K4me1**

rep1: GATCGGAAGAGCACACGTCTGAACTCCAGTCACCGATGT	1231626	3.940509670298028	TruSeq Adapter, Index 2

rep2: GATCGGAAGAGCACACGTCTGAACTCCAGTCACTAGCTT	1316788	4.335010476645132	TruSeq Adapter, Index 10 (100% over 39bp)

rep1 input: GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGA 1534234 4.698910117127738 TruSeq Adapter, Index 8 (100% over 39bp)

rep2 input:

GATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCA 918344 3.4431290053092054 TruSeq Adapter, Index 4 (100% over 39bp)

CATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCA 103628 0.38853041187418047 TruSeq Adapter, Index 4 (97% over 39bp)

In [121]:
%%bash
cd ../chip_sexual_organised/k4me1/
echo ">TrueSeq2" > TrueSeqk4me1.fa
echo "GATCGGAAGAGCACACGTCTGAACTCCAGTCACCGATGT" >> TrueSeqk4me1.fa
echo ">TrueSeq10" >> TrueSeqk4me1.fa
echo "GATCGGAAGAGCACACGTCTGAACTCCAGTCACTAGCTT" >> TrueSeqk4me1.fa
echo ">TrueSeq8" >> TrueSeqk4me1.fa
echo "GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGA" >> TrueSeqk4me1.fa
echo ">TrueSeq4" >> TrueSeqk4me1.fa
echo "GATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCA" >> TrueSeqk4me1.fa
less TrueSeqk4me1.fa

>TrueSeq2
GATCGGAAGAGCACACGTCTGAACTCCAGTCACCGATGT
>TrueSeq10
GATCGGAAGAGCACACGTCTGAACTCCAGTCACTAGCTT
>TrueSeq8
GATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGA
>TrueSeq4
GATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCA


### 3.2.1 h3k4me1_rep1

In [125]:
#%%bash
#cd ../chip_sexual_organised/k4me1/
#java -jar ~/software/Trimmomatic-0.39/trimmomatic-0.39.jar PE h3k4me1_rep1_R1.fastq.gz h3k4me1_rep1_R2.fastq.gz h3k4me1_rep1_R1_trimmomatic.fastq h3k4me1_rep1_R1_untrimmomatic.fastq h3k4me1_rep1_R2_trimmomatic.fastq h3k4me1_rep1_R2_untrimmomatic.fastq ILLUMINACLIP:TrueSeqk4me1.fa:2:30:10

In [127]:
#%%bash
#cd ../chip_sexual_organised/k4me1/
#fastqc h3k4me1_rep1_R1_trimmomatic.fastq h3k4me1_rep1_R2_trimmomatic.fastq

In [130]:
%%bash
cd ../chip_sexual_organised/k4me1/
cd h3k4me1_rep1_R1_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	h3k4me1_rep1_R1_trimmomatic.fastq
PASS	Per base sequence quality	h3k4me1_rep1_R1_trimmomatic.fastq
PASS	Per sequence quality scores	h3k4me1_rep1_R1_trimmomatic.fastq
FAIL	Per base sequence content	h3k4me1_rep1_R1_trimmomatic.fastq
PASS	Per sequence GC content	h3k4me1_rep1_R1_trimmomatic.fastq
PASS	Per base N content	h3k4me1_rep1_R1_trimmomatic.fastq
WARN	Sequence Length Distribution	h3k4me1_rep1_R1_trimmomatic.fastq
PASS	Sequence Duplication Levels	h3k4me1_rep1_R1_trimmomatic.fastq
PASS	Overrepresented sequences	h3k4me1_rep1_R1_trimmomatic.fastq
PASS	Adapter Content	h3k4me1_rep1_R1_trimmomatic.fastq


In [133]:
%%bash
cd ../chip_sexual_organised/k4me1/
cd h3k4me1_rep1_R2_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	h3k4me1_rep1_R2_trimmomatic.fastq
PASS	Per base sequence quality	h3k4me1_rep1_R2_trimmomatic.fastq
PASS	Per sequence quality scores	h3k4me1_rep1_R2_trimmomatic.fastq
FAIL	Per base sequence content	h3k4me1_rep1_R2_trimmomatic.fastq
PASS	Per sequence GC content	h3k4me1_rep1_R2_trimmomatic.fastq
PASS	Per base N content	h3k4me1_rep1_R2_trimmomatic.fastq
WARN	Sequence Length Distribution	h3k4me1_rep1_R2_trimmomatic.fastq
PASS	Sequence Duplication Levels	h3k4me1_rep1_R2_trimmomatic.fastq
PASS	Overrepresented sequences	h3k4me1_rep1_R2_trimmomatic.fastq
PASS	Adapter Content	h3k4me1_rep1_R2_trimmomatic.fastq


### 3.2.2 h3k4me1_rep2

In [137]:
#%%bash
#cd ../chip_sexual_organised/k4me1/
#java -jar ~/software/Trimmomatic-0.39/trimmomatic-0.39.jar PE h3k4me1_rep2_R1.fastq.gz h3k4me1_rep2_R2.fastq.gz h3k4me1_rep2_R1_trimmomatic.fastq h3k4me1_rep2_R1_untrimmomatic.fastq h3k4me1_rep2_R2_trimmomatic.fastq h3k4me1_rep2_R2_untrimmomatic.fastq ILLUMINACLIP:TrueSeqk4me1.fa:2:30:10

In [139]:
#%%bash
#cd ../chip_sexual_organised/k4me1/
#fastqc h3k4me1_rep2_R1_trimmomatic.fastq h3k4me1_rep2_R2_trimmomatic.fastq

In [142]:
%%bash
cd ../chip_sexual_organised/k4me1/
cd h3k4me1_rep2_R1_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	h3k4me1_rep2_R1_trimmomatic.fastq
PASS	Per base sequence quality	h3k4me1_rep2_R1_trimmomatic.fastq
PASS	Per sequence quality scores	h3k4me1_rep2_R1_trimmomatic.fastq
FAIL	Per base sequence content	h3k4me1_rep2_R1_trimmomatic.fastq
PASS	Per sequence GC content	h3k4me1_rep2_R1_trimmomatic.fastq
PASS	Per base N content	h3k4me1_rep2_R1_trimmomatic.fastq
WARN	Sequence Length Distribution	h3k4me1_rep2_R1_trimmomatic.fastq
PASS	Sequence Duplication Levels	h3k4me1_rep2_R1_trimmomatic.fastq
PASS	Overrepresented sequences	h3k4me1_rep2_R1_trimmomatic.fastq
PASS	Adapter Content	h3k4me1_rep2_R1_trimmomatic.fastq


In [147]:
%%bash
cd ../chip_sexual_organised/k4me1/
cd h3k4me1_rep2_R2_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	h3k4me1_rep2_R2_trimmomatic.fastq
PASS	Per base sequence quality	h3k4me1_rep2_R2_trimmomatic.fastq
PASS	Per sequence quality scores	h3k4me1_rep2_R2_trimmomatic.fastq
FAIL	Per base sequence content	h3k4me1_rep2_R2_trimmomatic.fastq
PASS	Per sequence GC content	h3k4me1_rep2_R2_trimmomatic.fastq
PASS	Per base N content	h3k4me1_rep2_R2_trimmomatic.fastq
WARN	Sequence Length Distribution	h3k4me1_rep2_R2_trimmomatic.fastq
PASS	Sequence Duplication Levels	h3k4me1_rep2_R2_trimmomatic.fastq
PASS	Overrepresented sequences	h3k4me1_rep2_R2_trimmomatic.fastq
PASS	Adapter Content	h3k4me1_rep2_R2_trimmomatic.fastq


### 3.2.3 input_rep1

In [149]:
#%%bash
#cd ../chip_sexual_organised/k4me1/
#java -jar ~/software/Trimmomatic-0.39/trimmomatic-0.39.jar PE input_rep1_R1.fastq.gz input_rep1_R2.fastq.gz input_rep1_R1_trimmomatic.fastq input_rep1_R1_untrimmomatic.fastq input_rep1_R2_trimmomatic.fastq input_rep1_R2_untrimmomatic.fastq ILLUMINACLIP:TrueSeqk4me1.fa:2:30:10

In [151]:
#%%bash
#cd ../chip_sexual_organised/k4me1/
#fastqc input_rep1_R1_trimmomatic.fastq input_rep1_R2_trimmomatic.fastq

In [154]:
%%bash
cd ../chip_sexual_organised/k4me1/
cd input_rep1_R1_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	input_rep1_R1_trimmomatic.fastq
PASS	Per base sequence quality	input_rep1_R1_trimmomatic.fastq
PASS	Per sequence quality scores	input_rep1_R1_trimmomatic.fastq
FAIL	Per base sequence content	input_rep1_R1_trimmomatic.fastq
PASS	Per sequence GC content	input_rep1_R1_trimmomatic.fastq
PASS	Per base N content	input_rep1_R1_trimmomatic.fastq
WARN	Sequence Length Distribution	input_rep1_R1_trimmomatic.fastq
PASS	Sequence Duplication Levels	input_rep1_R1_trimmomatic.fastq
PASS	Overrepresented sequences	input_rep1_R1_trimmomatic.fastq
PASS	Adapter Content	input_rep1_R1_trimmomatic.fastq


In [158]:
%%bash
cd ../chip_sexual_organised/k4me1/
cd input_rep1_R2_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	input_rep1_R2_trimmomatic.fastq
PASS	Per base sequence quality	input_rep1_R2_trimmomatic.fastq
PASS	Per sequence quality scores	input_rep1_R2_trimmomatic.fastq
FAIL	Per base sequence content	input_rep1_R2_trimmomatic.fastq
PASS	Per sequence GC content	input_rep1_R2_trimmomatic.fastq
PASS	Per base N content	input_rep1_R2_trimmomatic.fastq
WARN	Sequence Length Distribution	input_rep1_R2_trimmomatic.fastq
PASS	Sequence Duplication Levels	input_rep1_R2_trimmomatic.fastq
PASS	Overrepresented sequences	input_rep1_R2_trimmomatic.fastq
PASS	Adapter Content	input_rep1_R2_trimmomatic.fastq


### 3.2.4 input_rep2

In [161]:
#%%bash
#cd ../chip_sexual_organised/k4me1/
#java -jar ~/software/Trimmomatic-0.39/trimmomatic-0.39.jar PE input_rep2.R1.fastq.gz input_rep2.R2.fastq.gz input_rep2_R1_trimmomatic.fastq input_rep2_R1_untrimmomatic.fastq input_rep2_R2_trimmomatic.fastq input_rep2_R2_untrimmomatic.fastq ILLUMINACLIP:TrueSeqk4me1.fa:2:30:10

In [163]:
#%%bash
#cd ../chip_sexual_organised/k4me1/
#fastqc input_rep2_R1_trimmomatic.fastq input_rep2_R2_trimmomatic.fastq

In [166]:
%%bash
cd ../chip_sexual_organised/k4me1/
cd input_rep2_R1_trimmomatic_fastqc
less summary.txt

PASS	Basic Statistics	input_rep2_R1_trimmomatic.fastq
PASS	Per base sequence quality	input_rep2_R1_trimmomatic.fastq
PASS	Per sequence quality scores	input_rep2_R1_trimmomatic.fastq
FAIL	Per base sequence content	input_rep2_R1_trimmomatic.fastq
PASS	Per sequence GC content	input_rep2_R1_trimmomatic.fastq
PASS	Per base N content	input_rep2_R1_trimmomatic.fastq
WARN	Sequence Length Distribution	input_rep2_R1_trimmomatic.fastq
PASS	Sequence Duplication Levels	input_rep2_R1_trimmomatic.fastq
PASS	Overrepresented sequences	input_rep2_R1_trimmomatic.fastq
PASS	Adapter Content	input_rep2_R1_trimmomatic.fastq


In [14]:
%%bash
cd /hydra/chip_neiro/k4me1/
cd input_rep2_R2_trimmomatic_fastqc
less summary.txt

summary.txt: No such file or directory


# 4. Alignment

## 4.1 H3K27ac

Rep1 was aligned with bowtie2:

In [1]:
%%bash
cd ../chip_sexual_organised/k27ac
echo "#!/bin/bash" > k27ac_final_rep1_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq -2 X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq -p 12 -S X1-H3K27ace-_S12_L001_final.sam" >> k27ac_final_rep1_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq -2 X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq -p 12 -S X1-H3K27ace-_S12_L002_final.sam" >> k27ac_final_rep1_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq -2 X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq -p 12 -S X1-H3K27ace-_S12_L003_final.sam" >> k27ac_final_rep1_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq -2 X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq -p 12 -S X1-H3K27ace-_S12_L004_final.sam" >> k27ac_final_rep1_bwt.sh

echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-input-_S9_L001_R1_trimmomatic.fastq -2 X1-input-_S9_L001_R2_trimmomatic.fastq -p 12 -S X1-input-_S9_L001_final.sam" >> k27ac_final_rep1_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-input-_S9_L002_R1_trimmomatic.fastq -2 X1-input-_S9_L002_R2_trimmomatic.fastq -p 12 -S X1-input-_S9_L002_final.sam" >> k27ac_final_rep1_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-input-_S9_L003_R1_trimmomatic.fastq -2 X1-input-_S9_L003_R2_trimmomatic.fastq -p 12 -S X1-input-_S9_L003_final.sam" >> k27ac_final_rep1_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-input-_S9_L004_R1_trimmomatic.fastq -2 X1-input-_S9_L004_R2_trimmomatic.fastq -p 12 -S X1-input-_S9_L004_final.sam" >> k27ac_final_rep1_bwt.sh
chmod +x k27ac_final_rep1_bwt.sh
less k27ac_final_rep1_bwt.sh

#!/bin/bash
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-H3K27ace-_S12_L001_R1_trimmomatic.fastq -2 X1-H3K27ace-_S12_L001_R2_trimmomatic.fastq -p 12 -S X1-H3K27ace-_S12_L001_final.sam
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-H3K27ace-_S12_L002_R1_trimmomatic.fastq -2 X1-H3K27ace-_S12_L002_R2_trimmomatic.fastq -p 12 -S X1-H3K27ace-_S12_L002_final.sam
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-H3K27ace-_S12_L003_R1_trimmomatic.fastq -2 X1-H3K27ace-_S12_L003_R2_trimmomatic.fastq -p 12 -S X1-H3K27ace-_S12_L003_final.sam
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-H3K27ace-_S12_L004_R1_trimmomatic.fastq -2 X1-H3K27ace-_S12_L004_R2_trimmomatic.fastq -p 12 -S X1-H3K27ace-_S12_L004_final.sam
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 X1-input-_S9_L001_R1_tr

In [118]:
#%%bash
#cd ../chip_sexual_organised/k27ac
#nohup ./k27ac_final_rep1_bwt.sh > nohup_btwfinal.out 2>&1  </dev/null &

Rep2 and rep3 were aligned 

In [2]:
%%bash
cd ../chip_sexual_organised/k27ac
echo "#!/bin/bash" > k27ac_final_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 x1_h3k27ac_rep2.R1.fastq.gz -2 x1_h3k27ac_rep2.R2.fastq.gz -p 12 -S x1_h3k27ac_rep2_final.sam" >> k27ac_final_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 x1_input_rep2.R1.fastq.gz -2 x1_input_rep2.R2.fastq.gz -p 12 -S x1_input_rep2_final.sam" >> k27ac_final_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 x1_h3k27ac_rep3.R1.fastq.gz -2 x1_h3k27ac_rep3.R2.fastq.gz -p 12 -S x1_h3k27ac_rep3_final.sam" >> k27ac_final_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 x1_input_rep3_R1.fastq.gz -2 x1_input_rep3_R2.fastq.gz -p 12 -S x1_input_rep3_final.sam" >> k27ac_final_bwt.sh
chmod +x k27ac_final_bwt.sh
less k27ac_final_bwt.sh

#!/bin/bash
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 x1_h3k27ac_rep2.R1.fastq.gz -2 x1_h3k27ac_rep2.R2.fastq.gz -p 12 -S x1_h3k27ac_rep2_final.sam
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 x1_input_rep2.R1.fastq.gz -2 x1_input_rep2.R2.fastq.gz -p 12 -S x1_input_rep2_final.sam
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 x1_h3k27ac_rep3.R1.fastq.gz -2 x1_h3k27ac_rep3.R2.fastq.gz -p 12 -S x1_h3k27ac_rep3_final.sam
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 x1_input_rep3_R1.fastq.gz -2 x1_input_rep3_R2.fastq.gz -p 12 -S x1_input_rep3_final.sam


In [ ]:
#%%bash
#cd ../chip_sexual_organised/k27ac
#nohup ./k27ac_final_bwt.sh > nohup_btwfinal.out 2>&1  </dev/null &

## 4.2 H3K4me1

In [173]:
%%bash
cd ../chip_sexual_organised/k4me1
echo "#!/bin/bash" > k4me1_final_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 h3k4me1_rep1_R1_trimmomatic.fastq -2 h3k4me1_rep1_R2_trimmomatic.fastq -p 12 -S h3k4me1_rep1_final.sam" >> k4me1_final_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 input_rep1_R1_trimmomatic.fastq -2 input_rep1_R2_trimmomatic.fastq -p 12 -S input_rep1_final.sam" >> k4me1_final_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 h3k4me1_rep2_R1_trimmomatic.fastq -2 h3k4me1_rep2_R2_trimmomatic.fastq -p 12 -S h3k4me1_rep2_final.sam" >> k4me1_final_bwt.sh
echo "bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 input_rep2_R1_trimmomatic.fastq -2 input_rep2_R2_trimmomatic.fastq -p 12 -S input_rep2_final.sam" >> k4me1_final_bwt.sh
chmod +x k4me1_final_bwt.sh
less k4me1_final_bwt.sh

#!/bin/bash
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 h3k4me1_rep1_R1_trimmomatic.fastq -2 h3k4me1_rep1_R2_trimmomatic.fastq -p 12 -S h3k4me1_rep1_final.sam
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 input_rep1_R1_trimmomatic.fastq -2 input_rep1_R2_trimmomatic.fastq -p 12 -S input_rep1_final.sam
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 h3k4me1_rep2_R1_trimmomatic.fastq -2 h3k4me1_rep2_R2_trimmomatic.fastq -p 12 -S h3k4me1_rep2_final.sam
bowtie2 -x ~/sexual_genome_annotation_files/sexual_genome_annotation/final_dd_Smed_g4 -1 input_rep2_R1_trimmomatic.fastq -2 input_rep2_R2_trimmomatic.fastq -p 12 -S input_rep2_final.sam


In [ ]:
#%%bash
#cd ../chip_sexual_organised/k4me1
#nohup ./k4me1_final_bwt.sh > nohup_btwfinal.out 2>&1  </dev/null &

# FINNISHED